package/com.unity.formats.usd/Dependencies/USD.NET/serialization/TypeBinder.cs

```diff
@@ -68,6 +69,12 @@ public class TypeBinder
         Dictionary<string, string> typeNameMapping = new Dictionary<string, string>();
         Dictionary<Type, Dictionary<pxr.TfToken, Enum>> enumMaps = new Dictionary<Type, Dictionary<pxr.TfToken, Enum>>();
 
+        /// <summary>
+        /// USD types can be aliased to represent different roles float2/texcoord2, vec3/point/color and drive the interpretation
+        /// Add aliases if your target application need to map multiple USD types to a single type.
+        /// </summary>
+        public Dictionary<string, string> typeAliases = new Dictionary<string, string>();
```

Comment:
Is there a reason this needs to be exposed publicly? If not, I think it would be better to add a wrapper API around it for adding aliases.

Reply:
Not really ... I'll add a wrapper as advised.

Reply:
Done in https://github.com/Unity-Technologies/usd-unity-sdk/pull/260/commits/4e5e928f3618110e5ec30616b01f6e413329a25c